src/components/TimelineProposal.vue

```diff
@@ -39,8 +39,20 @@ watchEffect(() => {
     >
       <div>
         <div class="mb-2">
-          <Token :space="proposal.space" size="28" />
-          <span class="ml-2" v-text="proposal.space.name" />
+          <router-link
```

Comment:
Adding link is not relevant here because this component is only used when we are in a space page

Reply:
Right, this `TimelineProposal` is used on this page alone. Will remove here